LICENSE

```diff
@@ -114,13 +114,13 @@ The externally maintained libraries used by Node.js are:
     IN THE SOFTWARE.
   """
 
-- ICU, located at deps/icu-small, is licensed as follows:
+- ICU, located at deps/icu, is licensed as follows:
```

Comment:
The ICU license changes would indicate you have a version of ICU locally in `deps/icu` which is being used in preference to `deps/icu-small` in the Git source tree:
https://github.com/nodejs/node/blob/6608cf286d4d30bfca6f0480d91f7be957aad37a/tools/license-builder.sh#L35-L58

Reply:
That was indeed the case.
Rebased and regenerated.